File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/extending-word-documents-and-excel-workbooks-in-vsto-add-ins-at-run-time_4.vb
        Dim sheet As Microsoft.Office.Interop.Excel.Worksheet =
            Globals.ThisAddIn.Application.ActiveSheet
        If sheet.ListObjects.Count > 0 Then
            Dim listObject As Excel.ListObject = sheet.ListObjects(1)
            Dim vstoListObject As Microsoft.Office.Tools.Excel.ListObject =
                Globals.Factory.GetVstoObject(listObject)
        End If

File: OLSource1/v1/VS_officedev/codesnippet/CSharp/extending-word-documents-and-excel-workbooks-in-vsto-add-ins-at-run-time_4.cs
            Microsoft.Office.Interop.Excel.Worksheet sheet =
                Globals.ThisAddIn.Application.ActiveSheet;
            if (sheet.ListObjects.Count > 0)
            {
                Excel.ListObject listObject = 
                    sheet.ListObjects[1];
                Microsoft.Office.Tools.Excel.ListObject vstoListObject =
                    Globals.Factory.GetVstoObject(listObject);
            }